Epsilon.Host.Frontend/src/components/Competence/CompetenceProfile.vue

```diff
@@ -4,15 +4,17 @@
 			<tr>
 				<td />
 				<th
-					v-for="col of store.state.domain.columnsSet?.types"
+					v-for="col of store.state.domain.columnsSet?.types.sort((a: LearningDomainType, b: LearningDomainType) => (a?.order || 0) - (b?.order || 0))"
```

Comment:
```suggestion
					v-for="col of store.state.domain.columnsSet?.types.sort((a) => a.order ?? 0)"
```

Reply:
Als je dit zo schrijft dan krijg je errors met linting, aangezien het undefined is. Ook worden de columns niet gesorteerd.

Reply:
Then we will take a look at it tomorrow because at my side it just works.

Otherwise, we have still the possibility to do the ordering in the backend. It is maybe also a nicer approach.
Just thought about that. ;-/